src/driver/main.cpp

```diff
@@ -565,11 +568,42 @@ struct verify : command<verify>
         auto t = c.ct.get_target();
         auto m = c.parameters.generate(p, t, true, c.l.batch);
 
+
+        //TODO remove this and make the driver able to figure out datatype most used in the model
+        // then set the tolerances appropriately. Need to check here because c.to_fp16 only set
```

Comment:
I think an easy hack would be to look for dtype of convolution and/or GEMM. and based on that tolerances can be set.